.github/workflows/publish_precompiled.yml

```diff
@@ -0,0 +1,70 @@
+name: Publish Precompiled
+
+on:
+  push:
+    branches:
+      - main
+
+env:
+  ELIXIR_OPTS: "--warnings-as-errors"
+  ERLC_OPTS: "warnings_as_errors"
+  LANG: C.UTF-8
+  S3_BUCKET: s3.hex.pm
+
+jobs:
+  publish_precompiled:
+    strategy:
+      fail-fast: true
+      # Prevent each job edit builds.txt concurrently.
+      max-parallel: 1
+      matrix:
+        include:
+          - otp: 24
+            otp_version: 24.3
+          - otp: 25
+            otp_version: 25.0
+    runs-on: ubuntu-18.04
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          fetch-depth: 50
+
+      - uses: erlef/setup-beam@v1
+        with:
+          otp-version: ${{ matrix.otp_version }}
+          version-type: strict
+
+      - name: Build Elixir Release
+        run: |
+          make Precompiled.zip
+          mv Precompiled.zip main-otp-${{ matrix.otp }}.zip
+          shasum -a 1 main-otp-${{ matrix.otp }}.zip > main-otp-${{ matrix.otp }}.zip.sha1sum
+          shasum -a 256 main-otp-${{ matrix.otp }}.zip > main-otp-${{ matrix.otp }}.zip.sha256sum
+
+      - name: Update builds.txt
+        run: |
+          date=$(date -u '+%Y-%m-%dT%H:%M:%SZ')
+          build_sha256=$(cut -d ' ' -f 1 main-otp-${{ matrix.otp }}.zip.sha256sum)
+
+          aws s3 cp s3://${{ env.S3_BUCKET }}/builds/elixir/builds.txt builds.txt || true
+          touch builds.txt
```

Comment:
I'd remove ` || true` and `touch builds.txt` as it could obscure s3 network errors. It seems it is only useful for when there's no builds.txt in s3 already?

Reply:
Yes it is

Reply:
I changed to || touch builds.txt instead.:)

Reply:
If we got an intermittent network failure on `aws s3 cp` then we would create empty builds.txt whereas I think it is better to crash.

Reply:
Ahh, I see. I’ll take a look at it